### PR TITLE
Feature/bocm 277 get appeal details for inspector

### PR DESCRIPTION
**Pull request by mayya-bond-kc**
## Describe your changes

creates endpoint for inspector to get details of appeal

## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: apps/api/prisma/schema.prisma [hunk 1]
```diff
@@ -80,25 +85,27 @@ model ValidationDecision {
...
+  inGreenBelt                                           Boolean?
+  inOrNearConservationArea                              Boolean?
+  siteVisibleFromPublicLand                             Boolean?
+  siteVisibleFromPublicLandDescription                  String?
+  doesInspectorNeedToEnterSite                          Boolean?
+  doesInspectorNeedToEnterSiteDescription               String?
+  doesInspectorNeedToAccessNeighboursLand               Boolean?
+  doesInspectorNeedToAccessNeighboursLandDescription    String?
+  healthAndSafetyIssues                                 Boolean?
+  healthAndSafetyIsueesDescription                      String?
```
Comment (brpins) [inline apps/api/prisma/schema.prisma:103]: typo

File: apps/api/src/server/app/inspector/inspector.controller.js [hunk 2]
```diff
@@ -63,6 +63,132 @@ const getAppeals = async function(request, response) {
...
+		appellant: true, 
+		validationDecision: {
+			where: {
+				decision: 'complete',
+			},
+		},
+		lpaQuestionnaire: true,
+		appealDetailsFromAppellant: true
+	});
+	const formattedAppeal = {
```
Comment (brpins) [inline apps/api/src/server/app/inspector/inspector.controller.js:77]: Missing:
- appeal age
- address
- local planning department
- booked site visit data

File: apps/api/prisma/schema.prisma (hunk 1 above)
Comment (mayya-bond-kc) [reply apps/api/prisma/schema.prisma:103]: D:

File: apps/api/src/server/app/inspector/inspector.controller.js (hunk 2 above)
Comment (mayya-bond-kc) [reply apps/api/src/server/app/inspector/inspector.controller.js:77]: aha, this wasnt in the ticket to begin with. thanks for letting me know

**Review by brpins — APPROVED** [review]

**Pull request merged**